### ISSUE TITLE
add new command to list items

### ISSUE THREAD

**Issue by shenwei356** [shenwei356/csvtk#35]
Example data

```
dept   name    item
A      Tom     I_A
A      Tom     I_B
A      Jerry   I_A
B      Wei     I_B
```

What I want

```
$ csvtk xxx -t -f dept -v name
dept   name
A      Tom;Jerry
B      Wei

$ csvtk xxx -t -f item -v name
item   name
I_A    Tom;Jerry
I_B    Tome;Wei
```

```
$ csvtk xxx -t -f dept,name -v item
dept    name    item
A       Tom     I_A;I_B
A       Jerry   I_A
B       Wei     I_B
```

**Comment by likelet** [comment]
You can named it into `collapse`

**Comment by shenwei356** [comment]
@likelet Accepted.

Available in [v0.12.0-dev](https://github.com/shenwei356/csvtk/releases/tag/v0.12.0-dev) or later versions.
